### ISSUE TITLE
Too much Mortar Team Targets spawn at Dun Morogh

### ISSUE THREAD

**Issue by Gasan918** [slowtorta/turtlewow-bug-tracker#5420]
**Please, provide a clear description what the bug is:**

4, 5 or sometimes even 6 Mortar Team Targets are can be seen at Mortar Team spot. Isn't it too much?

**Steps to reproduce the behavior with as much detail as possible:**

1. Come to Mortar Team spot (40.8 66.8) and check

![image](https://github.com/slowtorta/turtlewow-bug-tracker/assets/96973359/bbbbb5a3-f9e9-4cc0-85a5-2215663e4889)
![image](https://github.com/slowtorta/turtlewow-bug-tracker/assets/96973359/c18a0abc-de2b-4489-9def-e15b41e18d2c)

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC: https://database.turtle-wow.org/?npc=11875

**Expected behavior. Describe how it should work:**

One or perhaps two targets.

**Comment by twowTurtReynolds** [comment]
Indeed, that top picture looks ridiculous. We need to see if this works correctly on latest vmangos and if not, how exactly it should look blizzlike.
